scripts/create_jekyll_ninjabuild.py

```diff
@@ -32,13 +33,21 @@ def scan_adoc(adoc_filename, apparent_filename):
 
 if __name__ == "__main__":
     index_json = sys.argv[1]
-    input_dir = sys.argv[2]
+    config_yaml = sys.argv[2]
+    input_dir = sys.argv[3]
     if not os.path.exists(input_dir):
         raise Exception("Error: {} doesn't exist".format(input_dir))
-    output_dir = sys.argv[3]
-    output_ninjabuild = sys.argv[4]
+    output_dir = sys.argv[4]
+    output_ninjabuild = sys.argv[5]
 
-    category_pages = set([('index.adoc', 'Raspberry Pi Documentation'), ('404.adoc', 'Raspberry Pi Documentation')])
```

Comment:
I'm so glad that I managed to get rid of the "niggle" of having these hardcoded page-title strings embedded inside these python build-scripts :slightly_smiling_face: